tests/hilti/types/integer/shifting-too-much.hlt

```diff
@@ -0,0 +1,53 @@
+# @TEST-EXEC-FAIL: hiltic -jd %INPUT 2>>output
+# @TEST-EXEC: btest-diff output
+#
+# @TEST-EXEC: hiltic -jd runtime-fail-1.hlt 2>>stderr; grep -q hilti::rt::Overflow stderr
+# @TEST-EXEC: hiltic -jd runtime-fail-2.hlt 2>>stderr; grep -q hilti::rt::Overflow stderr
```

Comment:
I think it would be valuable to capture the location of the runtime failure. WDYT about dropping these explicit invocations here and switching `@TEST-START-FILE`/`@TEST-END-FILE` to `@TEST-START-NEXT`?